magento/magento2
# Fix duplicated footer in Magento_Customer emails

### Description (*)

Email templates for change email, change email and password, password reset have duplicated `Thank you` note which is already in the footer:
```
<td class="footer">
    <p class="closing">{{trans "Thank you, %store_name" store_name=$store.frontend_name}}!</p>
</td>
```
and footer template is imported.

Fixing this will remove duplicated content in email.

### Related Pull Requests


### Fixed Issues (if relevant)

Couldn't find any issues, maybe there is some.

### Manual testing scenarios (*)

1. As a logged in user go to User Area - My Account Information section and change password / email.
2. Go to your email box and check those emails.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#28433: Fix duplicated footer in Magento_Customer emails
